openncp-client/manifests/openncp-client-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: openncp-client-service
+  namespace: openncp
+spec:
+  type: NodePort
+  ports:
+  - name: http
+    port: 7080
+    targetPort: 7080
+    nodePort: 30002
```

Comment:
we do not tyically define nodePorts.  Typically the default is a LoadBalancer service type, and it's up to each admin installing the system to chose how they will install.  for now, this is OK

Reply:
Will adjust this in a future PR!